src/components/UI/StyledDropZone/StyledDropZone.tsx

```diff
@@ -75,24 +83,34 @@ export const StyledDropZone: FunctionComponent<DropZoneProps> = ({
 
       {error && <p className="max-w-lg text-rose text-lg leading-none font-bold mb-2">Error</p>}
       {fileTypeError && (
-        <p className="max-w-lg text-rose text-lg leading-none mb-2" data-testid="invalid-file-error">
-          Incorrect file type selected, Only {dropzoneOptions.accept} are allowed
-        </p>
+        <ErrorMessage
+          id="invalid-file-error"
+          message={`Incorrect file type selected, Only ${dropzoneOptions.accept} are allowed`}
+        />
       )}
       {fileSizeError && (
-        <p className="max-w-lg text-rose text-lg leading-none mb-2" data-testid="file-size-error">
-          File size exceeds{" "}
-          {dropzoneOptions.maxSize ? ` ${dropzoneOptions.maxSize / FileUpload.BYTE_TO_MB_CONVERTION_RATE}` : ""}MB,
-          Please try again with a smaller file size.
-        </p>
+        <ErrorMessage
+          id="file-size-error"
+          message={`File size exceeds ${
+            dropzoneOptions.maxSize ? ` ${dropzoneOptions.maxSize / FileUpload.BYTE_TO_MB_CONVERSION_RATE}` : ""
+          }MB, Please try again with a smaller file size.`}
+        />
+      )}
+      {tooManyFilesError && (
+        <ErrorMessage
+          id="too-many-files-error"
+          message={`Upload too many documents at once, please upload a total of ${dropzoneOptions.maxFiles} document at a time.`}
+        />
       )}
       {fileErrors &&
         fileErrors.length > 0 &&
         fileErrors.map((formError, index: number) => {
           return (
-            <p key={index} className="max-w-lg text-rose text-lg leading-none mb-2" data-testid="file-error">
-              {formError instanceof Error ? "" : formError.instancePath} {formError.message}
-            </p>
+            <ErrorMessage
+              key={index}
```

Comment:
```suggestion
              key={`file-errors-${index}`}
```

Reply:
any reason for this change to the key attribute? it wont be display anywhere right?

Reply:
last resort -> https://reactjs.org/docs/lists-and-keys.html#keys, ok to ignore also

Reply:
ok updated